torch_xla/csrc/tensor_methods.h

```diff
@@ -1,4 +1,5 @@
-#pragma once
+#ifndef XLA_TORCH_XLA_CSRC_TENSOR_METHODS_H_
+#define XLA_TORCH_XLA_CSRC_TENSOR_METHODS_H_
```

Comment:
I may miss something here. I thought the c++ macro takes the form `#define identifier replacement`. So what's the replacement for the macro identifier `XLA_TORCH_XLA_CSRC_TENSOR_METHODS_H_`?